source/iface/group.rst

```diff
@@ -4,114 +4,316 @@
+  void parallel_for_work_item(range<Dimensions> logicalRange,
+                              const WorkItemFunctionT& func) const
+
+Launch the work-items for this work-group using a logical local range.
+The function object ``func`` is executed as if the kernel were invoked
+with ``logicalRange`` as the local range. This new local range is
+emulated and may not map one-to-one with the physical range.
+
+``logicalRange`` is the new local range to be used. This range can be
+smaller or larger than the one used to invoke the kernel. ``func`` is
+a function object type with a public member function
+``void F::operator()(h_item<Dimensions>)``
+representing the work-item computation.
+
+Note that the logical range does not need to be uniform across all
+work-groups in a kernel. For example the logical range may depend on
+a work-group varying query (e.g. ``group::get_linear_id``),
+such that different work-groups in the same kernel invocation
+execute different logical range sizes.
+
+This member function can only be invoked within a
+``parallel_for_work_group`` context.
 
 ``async_work_group_copy``
 =========================
 
 ::
 
-  template <typename dataT>
-  device_event async_work_group_copy(local_ptr<dataT> dest,
-    global_ptr<dataT> src, size_t numElements) const;
-  template <typename dataT>
-  device_event async_work_group_copy(global_ptr<dataT> dest,
-    local_ptr<dataT> src, size_t numElements) const;
-  template <typename dataT>
-  device_event async_work_group_copy(local_ptr<dataT> dest,
-    global_ptr<dataT> src, size_t numElements, size_t srcStride) const;
-  template <typename dataT>
-  device_event async_work_group_copy(global_ptr<dataT> dest,
-    local_ptr<dataT> src, size_t numElements, size_t destStride) const;
+  template <typename DataT>
+  device_event async_work_group_copy(local_ptr<DataT> dest,
+                                     global_ptr<DataT> src,
+                                     size_t numElements) const
 
-``wait_for``
-============
+  template <typename DataT>
+  device_event async_work_group_copy(global_ptr<DataT> dest,
+                                     local_ptr<DataT> src,
+                                     size_t numElements) const
+
+  template <typename DataT>
+  device_event async_work_group_copy(local_ptr<DataT> dest,
+                                     global_ptr<DataT> src,
+                                     size_t numElements, size_t srcStride) const
+
+  template <typename DataT>
+  device_event async_work_group_copy(global_ptr<DataT> dest,
+                                     local_ptr<DataT> src,
+                                     size_t numElements, size_t destStride) const
+
+All four ``async_work_group_copy`` functions listed above are deprecated
+in SYCL 2020. Has the same effect as the overload taking
+``decorated_local_ptr`` and ``decorated_global_ptr``
+except that the ``dest`` and ``src`` parameters are
+``multi_ptr#s with [code]#access::decorated::legacy``.
```

Comment:
Formatting is broken here

Reply:
Removed 's' with https://github.com/KhronosGroup/SYCL_Reference/pull/116/commits/03bb586a2631aa3ede22960fc77267458f093781

Reply:
This is still rendering with markup in it.  To clarify, the text in SYCL Reference currently renders as:

except that the `dest` and `src` parameters are `multi_ptr# with [code]#access::decorated::legacy`

The markup from the SYCL specification should not be included in the SYCL reference.  When the [spec fix PR](https://github.com/KhronosGroup/SYCL-Docs/pull/518) is merged, the SYCL specification will no longer show the markup such as `[code]` and `#`.  Those markup tags don't belong in the rendered text.

So for example, the above clip in the SYCL reference should instead look like:

except that the `dest` and `src` parameters are `multi_ptr` with `access::decorated::legacy`

Reply:
Thanks for detailed description...fixed with https://github.com/KhronosGroup/SYCL_Reference/pull/116/commits/d043aec22cb0671b66095bb468712c9cbfba2716